ItemManagementService/BusinessLayer/BrandBusinessLayer.cs

```diff
@@ -22,29 +22,9 @@ public BrandBusinessLayer(IBrandDataAccess BrandDataAccess)
         /// 
         /// </summary>
         /// <returns></returns>
-        public List<Brand> GetAllBrands()
+        public List<BrandModel> GetAllBrands()
         {
-            List<Brand> result = new List<Brand>();
-            Brand singleBrand = new Brand();
-
-            var brands = _brandDataAccess.GetBrands();
-
-            for (int i = 0; i < brands.Count; i++)
-            {
-                singleBrand.Id = brands[i].Id;
-                singleBrand.BrandName = brands[i].BrandName;
-                singleBrand.Notes = brands[i].Notes;
-                singleBrand.IsActive = brands[i].IsActive;
-                singleBrand.CreateUserName = brands[i].CreateUserName;
-                singleBrand.CreateDttm = brands[i].CreateDttm;
-                singleBrand.UpdateUserName = brands[i].UpdateUserName;
-                singleBrand.UpdateDttm = brands[i].UpdateDttm;
-
-                result.Add(singleBrand);
-                singleBrand = new Brand();
-            }
-
-            return result;
+            return _brandDataAccess.GetBrands().Select(x => MapBrandToBrandModel(x)).ToList(); 
```

Comment:
nice! :)

Reply:
Thanks, but not mine. Guys helped me to reach this out)